assets/templates/accounts/association_credits.html

```diff
@@ -5,28 +5,38 @@
 {% block tab_credits %} active{% endblock %}
 
 {% block details %}
-    <div class="row mb-3">
+    <div class="row">
         <div class="col-sm-6 col-md-8">Current balance: <strong>{{ balance|euro }}</strong></div>
         <div class="col-sm-6 col-md-4">
             <a href="{% url 'association_transactions_csv' association_name=association.slug %}"
                class="btn btn-secondary btn-block"><i class="fa fa-file-export"></i> Transactions CSV</a>
         </div>
     </div>
 
-    {# 'Table' header #}
-    <div class="row mb-1">
-        <div class="col-2"><strong>When</strong></div>
-        <div class="col-1"><strong>From/to</strong></div>
-        <div class="col-4"><strong>Sender or recipient</strong></div>
-        <div class="col-2"><strong>Amount</strong></div>
-        <div class="col-3"><strong>Description</strong></div>
-    </div>
-
-    {# Table rows #}
-    {% for transaction in object_list %}
-        {% include 'accounts/snippet_credit_history_association.html' %}
-    {% endfor %}
+    {# The actual table #}
+    <div class="row">
+        <div class="col-12 table-responsive">
```

Comment:
If you use the grid layout, then yes, content should only be place inside the columns, which is fine here. However what I meant is that the grid layout is not needed here since we don't actually need a grid and also not make use of any grid since we only have one column here.

The grid layout is just a tool to help you in creating more complex layouts with columns that'll work well across devices. Since we don't have multiple columns here we don't need it so the divs can just as well be removed, which was my original comment although it maybe wasn't very clear

Reply:
I'm still not sure whether I made myself fully clear: there is absolutely no difference between this:

```html
<div class="row">
    <div class="col">
        Content
    </div>
</div>
```

And this:

```html
Content
```

What's happening in the first instance is that the outer div has -15px negative margin and the inner div has 15px positive padding to undo the negative margin. You can try things for yourself here (use the Chrome or Firefox developer console) https://codepen.io/anon/pen/zXadaz

Now in the first screenshot I see that there's actually extra margin or padding on the sides of the table. That is maybe due to that the grid system is not correctly used here because the content should be inside the column, not merged with it. So foremost to make correct use of the grid system it should be like this:

```html
<div class="row">
    <div class="col-12">
        <div class="table-responsive">
            <table/>
        </div>
    </div>
</div>
```

But that is exactly the same as writing this:

```html
<div class="table-responsive">
    <table/>
</div>
```